source/includes/markdown/_25-app-components.html.md

```diff
@@ -303,6 +320,11 @@ _Note: An App Server must be hosted in order for Rule Actions to function. For a
 
 To see an example app server written for Rule Actions, see our [app-components-rule-action-example-app](https://github.com/Asana/app-components-rule-action-example-app) on GitHub.
 
+### UI Builder: Rule Action
+
+Visit the [UI Builder](https://app.asana.com/0/my-apps/response-builder/rule-action) to configure a Rule Action in an interactive test environment.
```

Comment:
the last part of the url is actually `rule-actions` (with an 's')

Reply:
Fixed!